packages/forms/src/UIForm/Widget/condition.js

```diff
@@ -1,18 +1,81 @@
 import get from 'lodash/get';
-import includes from 'lodash/includes';
+import findIndex from 'lodash/findIndex';
 
-function getValue(value, strategy) {
-	if (!strategy) {
+function toNumber(value) {
```

Comment:
if the value is not a string this act as identity function, wich is not expected for a toNumber function.

Reply:
any advice? throw an exception?